frontend/index.html

```diff
@@ -7,6 +7,11 @@
     <title>Maxitter | タイムライン</title>
   </head>
   <body>
+    <header>
+      <a href="">
```

Comment:
これだと現在いるページを再読み込みする挙動になっているので、リンク先としてトップページ (`/`) を指定するとベターかなと思いました～

```suggestion
      <a href="/">
```

Reply:
気づかなかった！